lib/lib_sms.php

```diff
@@ -25,26 +25,34 @@
 *   Array of contacts to be loaded.
 * @param int $language_id
 *   The language the contact must speak.  If zero, all languages will be returned.
-* @param bool $texting
-*   If true, the contact must support texting.
+* @param array $receives
+*   An array of boolean values, with keys 'calls', 'texts' and 'answered_alerts'.  If
+*   these are set to true, then only contacts that support receiving these will be
+*   returned.
+* @param bool $calls
```

Comment:
This parameter listing is erroneous. Was it intended?

Reply:
No, I forgot to remove the $calls parameter.  I'll push a fix.

Reply:
Actually, this was fixed in a later commit.  For some reason we're only viewing changes from 1 commit.  If you select "changes from all commits" then it is fixed!

Reply:
Apologies, user error.